feature_creation_scripts/antarctic_ocean_regions/antarctic_ocean_regions.py

```diff
@@ -0,0 +1,280 @@
+                                      (0, epsilon),
+                                      (-epsilon**2, -epsilon),
+                                      (-epsilon, minY),
+                                      (epsilon, minY)])
+
+    difference = poly.difference(wedge)
+
+    difference = shapely.ops.transform(stereo_to_lon_lat, difference)
+
+    x, y = difference.exterior.xy
+
+    plt.figure()
+    plt.plot(x, y)
+
+    fc = FeatureCollection()
+
+    geometry = shapely.geometry.mapping(difference)
+    # get rid of the wedge again by rounding the coordinates
+    geometry['coordinates'] = _round_coords(geometry['coordinates'])
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": "Contour {}".format(contourValue),
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean'},
+         "geometry": geometry})
+
+    return fc
+
+
+def make_rectangle(lon0, lon1, lat0, lat1, name, author, tags):
+    fc = FeatureCollection()
+
+    fc.add_feature(
+        {"type": "Feature",
+         "properties": {"name": name,
+                        "author": author,
+                        "object": 'region',
+                        "component": 'ocean',
+                        "tags": tags},
+         "geometry": {
+             "type": "Polygon",
+             "coordinates": [[[lon0, lat0],
+                              [lon1, lat0],
+                              [lon1, lat1],
+                              [lon0, lat1],
+                              [lon0, lat0]]]}})
+    return fc
+
+
+def split_rectangle(lon0, lon1, lat0, lat1, name, author, tags, fcContour):
+    fc = make_rectangle(lon0, lon1, lat0, lat1, name, author, tags)
+
+    fcDeep = fc.difference(fcContour)
+
+    props = fcDeep.features[0]['properties']
+    props['name'] = props['name'] + ' Deep'
+    props['tags'] = props['tags'] + ';Deep'
+    props['zmin'] = -1000.
+    props['zmax'] = -400.
+
+    fcShelf = fc.difference(fcDeep)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
+
+    fc.merge(fcDeep)
+    fc.merge(fcShelf)
+
+    return fc
+
+
+def main():
+    author = 'Xylar Asay-Davis'
+    timTags = 'Antarctic;Timmermann'
+    kusTags = 'Antarctic;Kusahara'
+
+    # make a geometric fieatures object that knows about the geometric data
+    # cache up a couple of directories
+    gf = GeometricFeatures('../../geometric_data')
+
+    bedmap2_bin_to_netcdf('bedmap2.nc')
+
+    fcContour700 = get_longest_contour(contourValue=-700., author=author)
+    fcContour800 = get_longest_contour(contourValue=-800., author=author)
+
+    fc = FeatureCollection()
+
+    fc.merge(split_rectangle(
+        lon0=-63., lon1=0., lat0=-80., lat1=-65., name='Weddell Sea',
+        author=author, tags=timTags, fcContour=fcContour800))
```

Comment:
@xylar: looking more closely at the region boundaries. I think the full Weddell Sea should extent more to the north and east, because we are cutting through the Weddell Gyre at 65S and not considering the extent east of 0. I think part of the gyre extent to at least 58S, and then to 20 or 30E. I can check more precise numbers before we settle.

Reply:
This would be a different region than the Weddell region considered in the Timmerman paper.

Reply:
Yeah, I'm somewhat hesitant to deviate from their definitions.  I could make the definitions of the Western and Eastern Weddell Seas whatever we think is best (since those I'm making up as I go along).  They already extend further east.  I can move the northern boundary to 58 S.

Reply:
oh I see, I didn't realize that the Weddell region came from the paper. I can check the Western and Eastern Weddell boundaries then, no problem, although probably not right now, if that's OK with you. Also, could we also merge the 2 regions and have a Weddell Gyre region or something like that?

Reply:
I expanded the Eastern and Western Weddell further north.